packages/types/src/types/ProvidersInventory.ts

```diff
@@ -0,0 +1,148 @@
+import { V1beta1Provider } from '../models/V1beta1Provider';
+
+/**
+ * Represents the inventory of providers, including their entities.
+ */
+export interface ProvidersInventory {
+  openshift?: OpenshiftInventory[] | null;
+  openstack?: OpenstackEntity[] | null;
+  ovirt?: OVirtInventory[] | null;
+  vsphere?: VSphereInventory[] | null;
+}
+
+/**
+ * General provider inventory
+ */
+export type ProviderInventory =
+  | OpenshiftInventory
+  | VSphereInventory
+  | OVirtInventory
+  | OpenstackEntity;
```

Comment:
`OpenStackInventory`?
1. the same suffix as others
2. it seems the wording used by the project is OpenStack not Openstack